docker-compose.yml

```diff
@@ -1,18 +1,23 @@
 version: '2'
 services:
   jarbas:
-    build:
-      context: .
-      dockerfile: Dockerfile
+    image: felipedemorais/auto-jarbas
     volumes:
-       - "staticfiles:/code/staticfiles"
+       - "assets:/code/staticfiles"
        - "./:/code"
+       - ".env:/code/.env"
     environment:
        - SECRET_KEY=lets-rock
```

Comment:
It's extremely unsafe to use this for production. As the focus is development environment this is not a problem right now… but I decided to let this note here so we can (later) take care of that and of `GOOGLE_STREET_VIEW_API_KEY` ; )